lib/carin_for_blue_button_test_kit/carin_search_test.rb

```diff
@@ -191,6 +191,7 @@ def run_include_search(search_params, param_value)
 
       skip_if returned_resources_resource_type.blank?, self.no_resources_message
 
+      values_found = []
       returned_resources_resource_type.each do |resource|
```

Comment:
This name "returned_resources_resource_type" is really confusing. This should be changed to simly "base_resources"